sycl/test/basic_tests/host_always_works.cpp

```diff
@@ -0,0 +1,23 @@
+// RUN: %clangxx -fsycl %s -o %t1.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out
```

Comment:
Please add the following line to cover accelerator as well:
// RUN: %ACC_RUN_PLACEHOLDER %t1.out

Reply:
Done.